add-news/index.md

```diff
@@ -0,0 +1,13 @@
+## How do I make a news post?
+
+News updates trigger follower notifications and are an essential part of publicizing your game. To create a news post for a particular game, go to your [developer dashboard](http://gamejolt.com/dashboard/) and select the game from under "Your Games". On the game's management page, visit the "News" tab and click "Add Game News".
+
+Here, you can give your post a title and write the post's content. News posts are written in Markdown, which is easy to use and allows you to do a number of cool things without having to write in HTML. For help with  Markdown, consult our [Markdown guide](http://help.gamejolt.com/markdown-games). 
+
```

Comment:
we should also include that if you post a bunch of news in a small amount of time, it will negate your score so you won't show on the latest news page. but maybe say it in a tone they'll understand like "if you post a bunch of news posts in a small amount of time, they will not appear on the home page to maintain game discover-ability and offer a range of game news" <-- ugh it's really poorly written but i'm hoping you get the idea to write something purdy.